types/mongodb/index.d.ts

```diff
@@ -89,69 +102,34 @@ export class ReadPreference {
     static isValid(mode: string): boolean;
 }
 
-//http://mongodb.github.io/node-mongodb-native/2.1/api/Server.html
+// http://mongodb.github.io/node-mongodb-native/2.2/api/Server.html
 export interface SocketOptions {
-    // Reconnect on error. default:false
-    autoReconnect?: boolean;
-    // TCP Socket NoDelay option. default:true
-    noDelay?: boolean;
-    // TCP KeepAlive on the socket with a X ms delay before start. default:0
-    keepAlive?: number;
-    // TCP Connection timeout setting. default 0
-    connectTimeoutMS?: number;
-    // TCP Socket timeout setting. default 0
-    socketTimeoutMS?: number;
-}
-
-//http://mongodb.github.io/node-mongodb-native/2.1/api/Server.html
+    autoReconnect       ?: boolean      ;// Reconnect on error. default:false
+    noDelay             ?: boolean      ;// TCP Socket NoDelay option. default:true
+    keepAlive           ?: number       ;// TCP KeepAlive on the socket with a X ms delay before start. default:0
+    connectTimeoutMS    ?: number       ;// TCP Connection timeout setting. default 0
+    socketTimeoutMS     ?: number       ;// TCP Socket timeout setting. default 0
+}
+
+// http://mongodb.github.io/node-mongodb-native/2.2/api/Server.html
 export interface ServerOptions {
```

Comment:
These are already fixed in the next commit that will be sent soon. There is an error in the MongosOptions, SocketOptions and ReplSetOptions as well.